boxes/contract-only/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@aztec/contract_box",
+  "description": "My Aztec contract",
+  "private": true,
+  "version": "0.1.0",
+  "type": "module",
+  "scripts": {
+    "compile": "cd src/contracts && ${AZTEC_NARGO:-aztec-nargo} compile",
```

Comment:
Contracts are cloned directly into `src/` not `src/contracts`

Reply:
This command was also failing because there is a `/` preceding the directory string in the Nargo.toml dependencies. eg `"/noir-projects/aztec-nr/aztec" ` should be `"noir-projects/aztec-nr/aztec" `

Reply:
I also added a slash after the `git` field. eg `"https://github.com/AztecProtocol/aztec-packages"` to `"https://github.com/AztecProtocol/aztec-packages/"`

Reply:
thanks, that's a bug in the script itself, I'll fix it! thanks